code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -138,6 +138,7 @@
 	name = "\improper Unica 6 auto-revolver"
 	desc = "A retro high-powered autorevolver typically used by officers of the New Russia military. Uses .357 ammo."
 	icon_state = "mateba"
+	var/illegal_tech = FALSE
```

Comment:
does this do anything?

Reply:
Yes, it removes the Mateba from the illegal tech list so when you deconstruct it you aren't getting illegal tech unlocked from it.

Reply:
this doesn't make any sense, you're defining a new var and then never referencing it anywhere

Reply:
Let me mess with this a bit. To my knowledge it's defined in the research module as "if this is false it isn't worth illegal tech", but I'm going off of someone else's info on this.

Reply:
I'm doing this wrong. Ignore this, I'm changing it.